lib/arduino_ci/arduino_backend.rb

```diff
@@ -235,5 +239,14 @@ def install_local_library(path)
       Host.symlink(src_path, destination_path)
       cpp_library
     end
+
+    private
+
+    def should_use_dry_run?
+      ret = capture_json("version")
+      version = ret[:json]["VersionString"]
+      match = /(\d)\.(\d+)\.(\d+)/.match(version)
```

Comment:
Use `Gem::Version` instead of regex to do this comparison:
https://stackoverflow.com/a/3064161

Reply:
Done